yaml/pois.yaml

```diff
@@ -163,9 +163,15 @@ filters:
     min_zoom: LEAST(zoom + 4.90, 18)
     output: {kind: snow_cannon}
   - filter:
-      leisure: [fitness_centre, gym, fitness_station]
+      any:
+        leisure: [fitness_centre, gym]
+        amenity: gym
     min_zoom: LEAST(zoom + 3.98, 17)
     output: {kind: fitness}
+  - filter:
+      leisure: fitness_station
+    min_zoom: LEAST(zoom + 3.98, 17)
+    output: {kind: fitness_station}
```

Comment:
I don't think so... at least, that's not what the tests said!

There's a big difference between a gym and a "push up" station in the park, so I think it makes sense to keep them separate.